internal/security/proxy/requestor.go

```diff
@@ -19,13 +19,15 @@ import (
 	"crypto/tls"
 	"net/http"
 	"time"
-
-	"github.com/edgexfoundry/edgex-go/internal"
 )
 
-func NewRequestor(skipVerify bool, timeout int) internal.HttpCaller {
+type Requestor interface {
```

Comment:
I apologize if I didn't make this clear enough before, but this bit of code is the part I was calling out before. What I would like to see for this PR is to not introduce this new type, and instead everywhere we use `Requestor` to instead use `internal.HttpCaller`.

Reply:
OK I think we understand it now. I feel more comfortable removing Requestor and keep internal.HttpCaller for consistence. will update the project.

Reply:
Thank you